### ISSUE TITLE
[JeuxVideo] Unable to extract config URL

### ISSUE THREAD

**Issue by thperret** [rg3/youtube-dl#5190]
Hi,
I got an error while trying to download from jeuxvideo.com with this url : http://www.jeuxvideo.com/chroniques-video/0004/00044715/speed-game-rayman-legends-wii-u-wiiu-duel-sur-rayman-legends-00123787.htm
Note that it does not happen with other url (at least for the few I tested)
Version is 2015.03.09

```bash
$ youtube-dl --verbose http://www.jeuxvideo.com/chroniques-video/0004/00044715/speed-game-rayman-legends-wii-u-wiiu-duel-sur-rayman-legends-00123787.htm
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['--verbose', 'http://www.jeuxvideo.com/chroniques-video/0004/00044715/speed-game-rayman-legends-wii-u-wiiu-duel-sur-rayman-legends-00123787.htm']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.03.09
[debug] Python version 2.7.6 - Linux-3.13.0-46-generic-x86_64-with-Ubuntu-14.04-trusty
[debug] exe versions: avconv 9.16-6, avprobe 9.16-6
[debug] Proxy map: {}
[JeuxVideo] speed-game-rayman-legends-wii-u-wiiu-duel-sur-rayman-legends: Downloading webpage
ERROR: Unable to extract config URL; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/YoutubeDL.py", line 649, in extract_info
    ie_result = ie.extract(url)
  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/extractor/common.py", line 275, in extract
    return self._real_extract(url)
  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/extractor/jeuxvideo.py", line 31, in _real_extract
    webpage, 'config URL')
  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/extractor/common.py", line 557, in _html_search_regex
    res = self._search_regex(pattern, string, name, default, fatal, flags, group)
  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/extractor/common.py", line 547, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
RegexNotFoundError: Unable to extract config URL; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

Thank you

**Comment by jaimeMF** [comment]
It will be fixed in the next version, thanks for the report.

**Comment by thperret** [comment]
Thank to you for the really quick fix!

**Comment by dstftw** [comment]
Fixed in youtube-dl 2015.03.15. See [our FAQ](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) on how to update.
